woocommerce/woocommerce-gateway-stripe
# fixed bug that would send stripe outdated data from blocks checkout

Fixes #2195

## Changes proposed in this Pull Request:
This PR fixes a bug that would send outdated data to stripe during blocks checkout. It adds billingData to the array of dependencies so no cached data will be used.

## Testing instructions
- Activate UPE
- Go to WooCommerce > Settings > Payment > Activate Boleto 
- Add any product to cart
- Go to Blocks checkout
- Fill the checkout fields. Set country as Botswana
- Select Boleto as payment method
- Click on place order
- The error "This payment method is not available on the selected country" will show up
- Change country to Brazil and a [valid address/zipcode](https://www.fakeaddressgenerator.com/World/Brazil_address_generator)
- The error "'BW' is not a valid country for a Brazilian address ('BR' required)" must not show up. 
- The Boleto modal must show up

The bug happened with other payment methods but it was harder to see it happening. It would require to debug and see the values stored inside the `billingData` inside `onCheckoutAfterProcessingWithSuccess`


**Post merge**
